cmd/ddev/cmd/dotddev_assets/commands/web/drush.example

```diff
@@ -6,3 +6,8 @@
 ## Example: "ddev drush uli" or "ddev drush sql-cli" or "ddev drush --version"
 
 drush $@
+
+# To use the site-installed drush directly (rather than running /usr/local/bin/drush),
+# you might want to execute it directly:
+#
+# /var/www/html/vendor/bin/drush $@
```

Comment:
I tend to use the relative path i.e. `vendor/bin/drush`. Not sure if that is just me or if other do that too.

Reply:
Just bare vendor/bin/rush doesn't work normally with drupal8+ project type because the working directory is /var/www/html/web, so it needs to be ../vendor/bin/drush in that case.

Reply:
if the site is using any fairly modern version of drush, then it should work. Its worked from above the web docroot for a couple years now. I know it didn't used to, but that hasn't been true for a long time now.

Reply:
Here's the problem:
```
$ ddev exec pwd
/var/www/html/web
```
When using `ddev exec` with a Drupal project, the working directory inside the container is the docroot (explicitly to make drush work on older versions of Drupal, so you're right there, it could be changed for drupal8+, but not in this PR). But `ddev exec`  on drupal executes in the docroot, meaning that the relative path to site-local drush is not vendor/bin/drush, it would have to be ../vendor/bin/drush, and that's pretty ambiguous. 

But I do agree that we could change the working directory in drupal8+